eduNEXT/xblock-filesmanager
# fix: allow open a file only when it has one file selected

### Description

This allows opening a file only when one file is selected; otherwise, the option doesn't appear


Before           |  Now
:-------------------------:|:-------------------------:
![](https://github.com/eduNEXT/xblock-filesmanager/assets/134975835/bb0b009e-f105-495c-b119-53ed9e94b2f9)  |  ![](https://github.com/eduNEXT/xblock-filesmanager/assets/134975835/79dc720b-a2e4-4494-ae9f-aef2260b0910)

### How to test it

- Check this branch`jv/fix-open-selected-file` in the XBlock files manager

1. use node 18
 ```
nvm use
```

2. install dependencies
```
yarn install 
```
3 run to see any change in the react code 
```
yarn dev 
```
------------------------------------------------------------------------------------------------------------------------------------------------
Now select a file  (click in the file) -> "Open file in a new tab"
![option-file-open](https://github.com/eduNEXT/xblock-filesmanager/assets/134975835/e08565e5-1a24-4ac8-b616-83e60ce0cb41)


If you select more than one file/folder the option  "Open file in a new tab" should not appear

![some-options-file](https://github.com/eduNEXT/xblock-filesmanager/assets/134975835/0af2464b-ee39-4ce9-876e-969700ae911d)



**Merge checklist:**
Check off if complete *or* not applicable:
